### ISSUE TITLE
Error when running from nvidia.dali.pipeline import Pipeline

### ISSUE THREAD

**Issue by aalugore** [NVIDIA/DALI#696]
Hi. I am building Dali in a conda (conda version 4.6.8) using the 'release_v0.7' branch on PPC. I have been able to build successfully but when i enter a python(v3.6.8) interpreter I get the following error:

>>> from nvidia.dali.pipeline import Pipeline
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/anaconda3/conda-bld/dali_1553547705117/_build_env/lib/python3.6/site-packages/nvidia/dali/__init__.py", line 18, in <module>                                                                                            
    from . import ops
  File "/opt/anaconda3/conda-bld/dali_1553547705117/_build_env/lib/python3.6/site-packages/nvidia/dali/ops.py", line 19, in <module>                                                                                                 
    from nvidia.dali import backend as b
  File "/opt/anaconda3/conda-bld/dali_1553547705117/_build_env/lib/python3.6/site-packages/nvidia/dali/backend.py", line 23, in <module>                                                                                             
    Init(OpSpec("CPUAllocator"), OpSpec("PinnedCPUAllocator"), OpSpec("GPUAllocator"))
RuntimeError: [/opt/anaconda3/conda-bld/dali_1553547705117/work/dali/util/cuda_utils.h:71] CUDA runtime api error "initialization error"

I verified the python version i built with and the one I am running with match at 3.6.8. Here is the versioning information for the rest of my build:
GCC - 7.2
Boost - 1.67.0
NVIDIA CUDA - 10.1
protobuf - 3.6.1
ffmpeg - 4.0
nvjpeglib - 10.1
cmake - 3.12.2
opencv - 3.4.2

I build using the following steps:
cmake -DBUILD_TENSORFLOW=ON \
      -DCUDA_TOOLKIT_ROOT_DIR=$CONDA_PREFIX \
      -DCUDA_rt_LIBRARY=$CONDA_PREFIX/powerpc64le-conda_cos7-linux-gnu/sysroot/usr/lib/librt.so \
      -DNVJPEG_ROOT_DIR=$CONDA_PREFIX/lib64/ \
      -DFFMPEG_ROOT_DIR=$CONDA_PREFIX/lib \
      -DJPEG_INCLUDE_DIR=$CONDA_PREFIX/libjpeg-turbo/lib \
      -DJPEG_LIBRARY=$CONDA_PREFIX/libjpeg-turbo/lib/libjpeg.so \
      -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX \
      -DCMAKE_AR=$CONDA_PREFIX/bin/powerpc64le-conda_cos7-linux-gnu-ar \
      -DCMAKE_CXX_COMPILER_AR=$CONDA_PREFIX/bin/powerpc64le-conda_cos7-linux-gnu-gcc-ar \
      -DCMAKE_C_COMPILER_AR=$CONDA_PREFIX/bin/powerpc64le-conda_cos7-linux-gnu-gcc-ar \
      -DCMAKE_LINKER=$CONDA_PREFIX/bin/powerpc64le-conda_cos7-linux-gnu-gcc-ld \
      -DCMAKE_RANLIB=$CONDA_PREFIX/bin/powerpc64le-conda_cos7-linux-gnu-gcc-ranlib \
      -DCMAKE_CXX_COMPILER_RANLIB=$CONDA_PREFIX/bin/powerpc64le-conda_cos7-linux-gnu-gcc-ranlib \
      -DCMAKE_C_COMPILER_RANLIB=$CONDA_PREFIX/bin/powerpc64le-conda_cos7-linux-gnu-gcc-ranlib \
      -DCUDA_CUDA_LIBRARY=$CONDA_PREFIX/lib/stubs/libcuda.so \
      ..

make -j"$(nproc --all)" install

pip install --no-deps dali/python

Any ideas? Am I working with any incompatible version numbers?

**Comment by JanuszL** [comment]
Hi,
What do you get from:
```
pip show nvidia-dali
```
I guess it may be not properly installed in your env. Have you tried to install prebuild package (it can tell us if this is conda env or installing process when you building from the scratch)?

**Comment by aalugore** [comment]
Hi. So as it turned out, the system I was working on was having driver initialization issues. It crashed last night and after a reboot, the import statements worked. However, I have a follow up question/issue relating to #591 After I create the conda package and attempt to install it, I get the "No module named 'nvidia'" error. It was only after I forced my conda build to fail at the end of my build.sh script, entered the broken build's envtironment via conda activate _build_env, and entered a python interpreter from inside that env that I was able to get the import statement to work. So my question is, after building DALI, which artifacts/files/binaries do I need to make sure get put into my conda package? As of right now, after building and then manually unpacking the resulting tar.bz2 file, I see the only thing in there is the info metadata directory. I could not find any DALI documentation that detailed what generated files are needed in order to use it.
